src/components/AlertBar/AlertBar.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import { AlertBar, AlertStack } from '@dhis2/ui'
+import PropTypes from 'prop-types'
+
+import { sGetAlertMessage } from '../../reducers/alert'
+import { acClearAlertMessage } from '../../actions/alert'
+
+export const Alert = ({ message, onClose }) =>
+    message ? (
+        <AlertStack>
```

Comment:
Not sure we need `AlertStack` if we only have 1 message at a time...

Reply:
The AlertStack provides the positioning, so we need it.

Reply:
ok, i should probably update DV then.